benchmarks/Dapper.Tests.Performance/Dapper.Tests.Performance.csproj

```diff
@@ -3,6 +3,7 @@
     <AssemblyName>Dapper.Tests.Performance</AssemblyName>
     <Description>Dapper Core Performance Suite</Description>
     <OutputType>Exe</OutputType>
+    <TargetFrameworks>net462;net5.0;net8.0</TargetFrameworks>
```

Comment:
not sure we want to change this - what's the intent here?

Reply:
you're right, removed